torch/ao/nn/quantized/reference/modules/utils.py

```diff
@@ -98,13 +120,109 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
             state_dict, prefix, local_metadata, False,
             missing_keys, unexpected_keys, error_msgs)
 
+def _quantize_weight_decomposed(
+        weight: torch.Tensor,
+        weight_qscheme: torch.qscheme,
+        weight_dtype: torch.dtype,
+        weight_scale: torch.Tensor,
+        weight_zero_point: torch.Tensor,
+        weight_axis: int):
+    # TODO: get the quant_min and quant_max from activation_post_process
```

Comment:
is this for before land or for later?

Reply:
this is for later, we don't have this for all observers right now I think

Reply:
I think we should probably have a BE task to make quant_min/quant_max a required field